packages/rln/tsconfig.dev.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig",
+  "compilerOptions": {
+    "module": "ESNext",
+    "moduleResolution": "Bundler",
+    "noEmit": true
+  }
```

Comment:
the original tsconfig already has the same compiler options, we don't need to rewrite